_posts/2024-06-20-pyos-guide-to-scipy.md

```diff
@@ -0,0 +1,84 @@
+* Use PyVista to create 3D visualizations from a variety of datasets in common formats.
+* Overview the classes and data structures of PyVista with real-world examples.
+* Be familiar of the various filters and features of PyVista.
+* Know which Python libraries are used and can be used by PyVista (meshio, trimesh etc).
+* We see this tutorial catering to anyone who wants to visualize data in any domain, and this ranges from basic Python users to advanced power users.
+
+### [From RAGS to riches: Build an AI document inquiry web-app](https://cfp.scipy.org/2024/talk/W3ZJWG/)
+[Pavithra Eswaramoorthy](https://cfp.scipy.org/2024/speaker/QGMGFB/), a Developer Advocate at [Quansight](https://quansight.com/), is teaming up with [Dharhas Pothina](https://cfp.scipy.org/2024/speaker/EKHUEY/) and [Andrew Huang](https://cfp.scipy.org/2024/speaker/QSLXKW/) to cover how to use RAG to build document-inquiry chat systems using different commercial and locally running LLMs. The topics we’ll cover include:
+* **Introduction to RAG**, how it works and interacts with LLMs, and Ragna - a framework for RAG orchestration
+* **Creating and optimizing a basic chat function** that uses popular LLMs (like GPT) answers questions about your documents, using a Python API in Jupyter Notebooks
+* **Running a local LLM on GPUs** on the provided platform, and comparing its performance to commercial LLMs
+* **Introduction to Panel**, by creating a basic chat UI for Ragna using Panel’s ChatBox widget
+* **Building and deploying a Panel-based web-app**, which extends the basic chat UI and includes more application components
+
+By the end of this tutorial, you will have an understanding of the fundamental components that form a RAG model, and practical knowledge of open source tools that can help you or your organization explore and build on your own applications. This tutorial is designed to enable enthusiasts in our community to explore an interesting topic using some beginner-friendly Python libraries.
+
+By the end of this tutorial, you will have an understanding of the fundamental components that form a RAG model, and practical knowledge of open source tools that can help you or your organization explore and build on your own applications. This tutorial is designed to enable enthusiasts in our community to explore an interesting topic using some beginner-friendly Python libraries.
+
+### [Create Your First Python Package: Make Your Python Code Easier to Share and Use](https://cfp.scipy.org/2024/talk/QT9GBY/)
+Led by pyOpenSci’s Executive Director and Founder, [Leah Wasser](https://github.com/lwasser), in this tutorial session you’ll learn how to:
```

Comment:
Can you please add Isabel and Jeremiah to this? i've added them both to the landing page as well. 

Isabel is an editor as well / emeritus eic. jeremiah - is leading work on our packaging guide (among other things!).